package.json

```diff
@@ -34,10 +34,10 @@
     "storybook": "yarn workspace @metamask/storybook-react storybook",
     "storybook:android": "yarn workspace @metamask/storybook-react-native android",
     "storybook:ios": "yarn workspace @metamask/storybook-react-native ios",
-    "test": "yarn test:scripts --silent --collectCoverage=false --reporters=jest-silent-reporter && yarn test:packages",
+    "test": "NODE_OPTIONS=--experimental-vm-modules yarn test:scripts --silent --collectCoverage=false --reporters=jest-silent-reporter && yarn test:packages",
     "test:clean": "yarn workspaces foreach --all --parallel --verbose run test:clean && yarn test",
-    "test:packages": "yarn test:verbose --silent --collectCoverage=false --reporters=jest-silent-reporter",
-    "test:scripts": "yarn jest --config ./jest.config.scripts.js --silent",
+    "test:packages": "NODE_OPTIONS=--experimental-vm-modules yarn test:verbose --silent --collectCoverage=false --reporters=jest-silent-reporter",
+    "test:scripts": "NODE_OPTIONS=--experimental-vm-modules yarn jest --config ./jest.config.scripts.js --silent",
```

Comment:
Added `NODE_OPTIONS=--experimental-vm-modules` flag to test scripts to fix Jest ESM import errors.

This change was necessary because:
- Recent versions of our dependencies (particularly Prettier) use ESM modules
- Jest requires this flag to properly handle dynamic imports in an ESM context
- Without this flag, tests were failing with `ERR_VM_DYNAMIC_IMPORT_CALLBACK_MISSING_FLAG` error

This is a standard solution recommended by the Jest team for projects using ESM modules. The flag enables Node.js experimental support for ES modules in the VM context.